packages/gatsby-plugin-less/package.json

```diff
@@ -8,14 +8,19 @@
     "watch": "babel -w src --out-dir . --ignore __tests__"
   },
   "keywords": [
-    "gatsby"
+    "gatsby",
+    "less"
   ],
-  "author": "Kyle Mathews &lt;mathews.kyle@gmail.com&gt;",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "license": "MIT",
   "devDependencies": {
     "babel-cli": "^6.24.1"
   },
   "dependencies": {
-    "babel-runtime": "6.26.0"
+    "babel-runtime": "6.26.0",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "gatsby-1-config-css-modules": "^1.0.1",
+    "less": "^2.7.2",
+    "less-loader": "^4.0.5"
```

Comment:
This version is technically webpack v2 only, it's probably a coincidence that it works

Reply:
I see. Looking at the [release history](https://github.com/webpack-contrib/less-loader/releases), v3.0.0 says:
> There are no breaking changes for webpack 1 with 3.0.0, but future release won't be tested against webpack 1

So I think it's safe to go back to 3.0.0 at least, unless you think I should go further back (to 2.2.3)?